src/simulations/action_translator.py

```diff
@@ -341,8 +370,27 @@ def actions_to_env_vars(self, loads, home_vars, action, date, time_step):
         # loads: flex_cons, tot_cons_loads
         # home_vars: netp, bool_flex, tot_cons
         # bool_penalty
+        # flexible_q_car
+
+        return loads, home_vars, bool_penalty, flexible_q_car
+
+    def _calculate_flexible_q_car(self, indiv_flexible_store_action, indiv_flexible_q_car_action):
+        if indiv_flexible_q_car_action > 0:
+            charge = indiv_flexible_store_action
+            max_q_car_import = np.sqrt(self.max_apparent_power_car**2 - charge**2)
+            indiv_flexible_q_car = \
+                indiv_flexible_q_car_action * (max_q_car_import - self.min_q_car_import) \
+                + self.min_q_car_import
+        elif indiv_flexible_q_car_action < 0:
+            discharge = indiv_flexible_store_action
+            max_q_car_export = - np.sqrt(self.max_apparent_power_car**2 - discharge**2)
+            indiv_flexible_q_car = \
+                - abs(self.min_q_car_export
+                      - indiv_flexible_q_car_action * abs(self.min_q_car_export - max_q_car_export))
+        else:
+            indiv_flexible_q_car = 0
```

Comment:
Could this whole function simply be the below? (if `min_q_car_export =0`)
`q = q_action * np.sqrt(self.max_apparent_power_car**2 - ds**2)`

if we have `min_q_car_export > 0` then it would make no sense to have `q_action < 0` right?